kafka-connect-s3/pom.xml

```diff
@@ -197,6 +197,53 @@
                             </dependency>
                         </dependencies>
                     </plugin>
+                    <plugin>
+                        <groupId>io.confluent</groupId>
+                        <version>0.9.0</version>
+                        <artifactId>kafka-connect-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>kafka-connect</goal>
+                                </goals>
+                                <configuration>
+                                    <title>Kafka Connect Storage Cloud</title>
+                                    <sourceUrl>https://github.com/confluentinc/kafka-connect-storage-cloud</sourceUrl>
+                                    <documentationUrl>https://docs.confluent.io/current/connect/connect-storage-cloud/kafka-connect-s3/docs/index.html</documentationUrl>
+                                    <description>
+                                        The S3 connector, currently available as a sink, allows you to export data from Kafka topics to S3 objects in either Avro or JSON formats. In addition, for certain data layouts, S3 connector exports data by guaranteeing exactly-once delivery semantics to consumers of the S3 objects it produces.
+
+                                        Being a sink, the S3 connector periodically polls data from Kafka and in turn uploads it to S3. A partitioner is used to split the data of every Kafka partition into chunks. Each chunk of data is represented as an S3 object, whose key name encodes the topic, the Kafka partition and the start offset of this data chunk. If no partitioner is specified in the configuration, the default partitioner which preserves Kafka partitioning is used. The size of each data chunk is determined by the number of records written to S3 and by schema compatibility.
+                                    </description>
+
+                                    <supportProviderName>Confluent, Inc.</supportProviderName>
+                                    <supportSummary>Confluent supports the S3 sink connector alongside community members as part of its Confluent Platform open source offering.</supportSummary>
+                                    <supportUrl>https://docs.confluent.io/current/</supportUrl>
+                                    <supportLogo>logos/confluent.png</supportLogo>
```

Comment:
Either this element should be removed for now, or a `logos/` directory should be created with the `confluent.png` image in it.